src/site/content/en/learn/css/conclusion/index.md

```diff
@@ -35,4 +35,7 @@ To continue to build your CSS skillset, here are some additional resources to ex
 
 ## Brought to you by
 
-{% include 'partials/author-stack.njk' %}
+<!--lint disable-->
```

Comment:
Lint complains here, that you use template tags inside an .md file. This is a hint for you as a developer that you're doing something unusual, and therefore you should have a very strong reason to do so.

Also, you're repeating the same code over 4 files. Again a strong hint that this snippet of code is in the wrong place. 

Maybe it's worth moving this part to the njk template, instead of repeating code in .md files.